dist/map-panel.js

```diff
@@ -152,6 +152,7 @@ Fliplet.InteractiveMap.component('map-panel', {
         }
       });
       window.filePickerProvider.then(function (result) {
+        result.data[0].url = result.data[0].url + '?size=large';
```

Comment:
What if the URL already contains a `?` query character?

Reply:
@tonytlwu I have tested with different images, already in the folders and with newly uploaded, and none would come with a `?` query character.

However I still think this entire fix should be done in the backend. The backend should return me the file data with the `url` containing the `?size=large` since it will add it anyway after saving.

cc @squallstar

Reply:
@hcarneiro I think your fix should be less error prone by including the just in case scenarios. This includes:

1. What if URL already includes a `?`. The fact that it doesn't include one now doesn't mean it won't at some point in the short term future. A bit of coding to help prevent errors would definitely help everyone.
1. What if URL already includes `size=large` query? If and when we implement backend changes after you've deployed the frontend change, it would help prevent errors.

Because Growth and Product work on different priorities, it's important that @hcarneiro try to work without dependency on Product but make the code as future proof as possible.

Reply:
@tonytlwu How do you propose I make that error proof?
Using `RegExp` to check the URL?

Reply:
Partially, yes. `RegExp` should help. You can also use `.split()` and loops.

Reply:
@squallstar If you have code patterns that @hcarneiro can use, feel free to share.